.github/workflows/release.yml

```diff
@@ -22,4 +22,7 @@ jobs:
       # ensure gem can be built and installed, push to rubygems.org
       - run: gem build rubocop-eightyfourcodes
       - run: gem install *.gem
-      - run: gem push *.gem
+      - run: |
+          ruby -S gem exec sigstore-cli:0.2.1 sign *.gem --bundle attestation.sigstore.json
```

Comment:
I'm curious... why did you do `ruby -S` here, versus all the other calls you do `gem exec` by itself?

Reply:
Stumbled across it while researching https://github.com/dependabot/dependabot-core/pull/12025 😄